modules/core/package.json

```diff
@@ -69,6 +70,9 @@
     "superagent-proxy": "^1.0.3"
   },
   "devDependencies": {
+    "@babel/cli": "^7.4.4",
```

Comment:
Why are we adding babel?

Reply:
eh, algosdk won't build without it, we might need to re-factor algosdk-bitgo

Reply:
FACTORED